packages/ui/src/components/DateInput/__stories__/Template.tsx

```diff
@@ -1,12 +1,12 @@
 import type { StoryFn } from '@storybook/react'
-import type { ComponentProps } from 'react'
 import { DateInput } from '..'
 
-export const Template: StoryFn<ComponentProps<typeof DateInput>> = props => (
+export const Template: StoryFn<typeof DateInput> = props => (
   <DateInput {...props} />
 )
 
 Template.args = {
   label: 'Date Input',
   required: true,
+  placeholder: 'YYYY-MM-DD',
```

Comment:
Any reasons why the date start with the year? We usually go by day/months/years

Reply:
No particular reason, I simply copied the example in Figma haha

Reply:
Ah ok it's probably wrong then you can go with the usual day/months/years 😄